po/fr.po

```diff
@@ -26909,847 +26871,947 @@ msgid "set the list of administrators for GROUP"
 "and print all timestamps as UNIX timestamps"
 msgstr ""
+"Ne pas fusionner l’identifiant utilisateur primaire et la clé primaire dans "
+"les listes avec --with-colon, et afficher tous les horodatages au format UNIX"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:180
 msgid ""
 "Changes the behaviour of some commands. This is like --dry-run but different"
 msgstr ""
+"Change le comportement de certaines commandes ; similaire à, quoique "
+"différent de, --dry-run"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:181
 msgid "Display the session key used for one message"
-msgstr ""
+msgstr "Afficher la clé de session utilisée pour un message"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:182
 msgid "Don't use the public key but the specified session key"
-msgstr ""
+msgstr "Utiliser la clé de session spécifiée au lieu de la clé publique"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:183
 #: /tmp/fish/implicit/share/completions/gpg.fish:185
 msgid "Prompt for an expiration time"
-msgstr ""
+msgstr "Demander une date d’expiration"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:184
 #: /tmp/fish/implicit/share/completions/gpg.fish:186
 msgid "Do not prompt for an expiration time"
-msgstr ""
+msgstr "Ne pas demander une date d’expiration"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:187
 msgid ""
 "Don't look at the key ID as stored in the message but try all secret keys in "
 "turn to find the right decryption key"
 msgstr ""
+"Ne pas rechercher l’identifiant de clé tel qu’indiqué dans le message ; "
+"essayer à la place toutes les clés privées pour trouver la bonne"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:188
 msgid ""
 "Enable a mode in which filenames of the form -&n, where n is a non-negative "
 "decimal number, refer to the file descriptor n and not to a file with that "
 "name"
 msgstr ""
+"Activer un mode dans lequel les noms de fichiers au format -&n, pour tout n "
+"entier naturel, font référence au descripteur de fichier n et pas à un "
+"fichier ainsi nommé"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:189
 msgid "Sets up a named group, which is similar to aliases in email programs"
 msgstr ""
+"Paramètre un groupe nommé, similaire aux alias pour les programmes d’email"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:190
-#, fuzzy
 msgid "Remove a given entry from the --group list"
-msgstr "Enlever un fichier du référentiel"
+msgstr "Supprimer une entrée de la liste --group"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:191
-#, fuzzy
 msgid "Remove all entries from the --group list"
-msgstr "Enlever un fichier du référentiel"
+msgstr "Supprimer toutes les entrées de la liste --group"
 
 #: /tmp/fish/implicit/share/completions/gpg.fish:192
 msgid ""
 "Don't change the permissions of a secret keyring back to user read/write only"
-msgstr ""
+msgstr "Ne pas modifier à 400 les permissions d’un trousseau privé"
```

Comment:
Maybe better not assume everybody is familiar with numeric permissions?

Reply:
Well, I don't know if a native english speaker finds this sentence to be clear, but, exactly translated, the sentence would be *Ne pas modifier les permissions d’un trousseau privé vers un accès en lecture et écriture au seul utilisateur*. The problem is that this sentence is verbose and obscure for a french speaker, which has to re-process the sentence upon reading to correctly associate the last syntagma to the correct one among those before it; the octal translation allows a way clearer syntax.

Reply:
I understand your issue with the verbosity and complexity of the translation, but my point still stands: the octal translation only allows a clearer syntax **if every single user of fish is familiar with octal syntax for access rights**. I for one, despite using the command line extensively for the last 10 years, am **not** able to translate octal codes for access to their real world meaning, so I find that assumption flawed from the start.

Reply:
Ok, then let's go with the verbose one.